posts/2022-03-15-ga-release-post.adoc

```diff
@@ -0,0 +1,155 @@
+In link:{url-about}[Open Liberty] 22.0.0.3:
+
+* <<sql, Allow SQL Operations to be retried in transaction recovery logs>>
+
+Along with the new features and functions added to the runtime, we’ve also made <<guides, updates to our guides>>.
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A22003+label%3A%22release+bug%22+[22.0.0.3].
+
+[#run]
+== Run your apps using 22.0.0.3
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>22.0.0.3</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[22.0.0.3,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+[#sql]
+=== Allow SQL Operations to be retried in transaction recovery logs
+
+In Liberty, transaction services are implicitly activated when you specify features that require the use of transactions; for example, `jpa-2.0`, `jdbc-4.0`, and `wasJmsServer-1.0`.
+
+When Liberty detects a failure in writing to its transaction recovery logs, it will invalidate the logs and disallow further global transactions. In order to do further transactional work, the customer must restart the server. This means applications have to be reloaded when the server restarts. This can take a considerable amount of time for some applications.
+
+Under certain circumstances, such as when the recovery log failure is caused by transient database connection errors, the condition can be handled by retrying the database operation thus avoiding the need to shutdown the server and reload the apps.
+
+A new transaction service configuration attribute has been introduced to the transaction stanza in the Liberty `server.xml` file that allows these transaction log SQL operations to be retried. It is named `enableLogRetries` and should be set to `true` to enable retries. For example,
+
+[source,xml]
+----
+<transaction
+dataSourceRef="tranlogDataSource"
+recoverOnStartup="true"
+waitForRecovery="false"
+heuristicRetryInterval="10"
+enableLogRetries="true"
+/>
+----
+
+By default the new functionality is disabled.
+
+For more information on the storing or Transaction Recovery Logs in an `RDBMS`, please refer to link:https://openliberty.io/docs/22.0.0.2/transaction-service.html#log[Transaction log management] doc.
```

Comment:
use `latest` in the docs link unless you are targeting a specific version:

link:https://openliberty.io/docs/latest/transaction-service.html#log[Transaction log management]
or use relative URL
link:{url-prefix}/docs/latest/transaction-service.html#log[Transaction log management]